### PR TITLE
Delete uncessary config fields

### PR DESCRIPTION
**Pull request by parnurzeal**
To reduce a confusion on deployment

### CODE REVIEW

File: config/default.template.json [hunk 1]
```diff
@@ -5,8 +5,6 @@
   "mongodb": "mongodb://mongodb:27017/youpin-dev",
   "public": "../public/",
   "auth": {
-    "successRedirect": "your url for success login",
-    "failureRedirect": "your url for fail login",
```
Comment (supasate) [inline config/default.template.json:9]: `successRedirect` and `failureRedirect` are used with facebook authentication

File: config/production.template.json [hunk 2]
```diff
@@ -4,8 +4,6 @@
   "mongodb": "mongodb://mongodb:27017/youpin",
   "public": "../public/",
   "auth": {
-    "successRedirect": "your url for success login",
-    "failureRedirect": "your url for fail login",
```
Comment (supasate) [inline config/production.template.json:8]: `successRedirect` and `failureRedirect` are used with facebook authentication

Comment (parnurzeal) [reply config/production.template.json:8]: That's what I thought too but I didn't see anyone setting it yet on production...
So, I thought they were not needed.

Anyway, added them back. PTAL.

**Comment by supasate** [comment]
LGTM

**Pull request merged**